lib/trento_web/controllers/v1/activity_log_controller.ex

```diff
@@ -60,7 +62,12 @@ defmodule TrentoWeb.V1.ActivityLogController do
     ]
 
   def get_activity_log(conn, params) do
-    with {:ok, activity_log_entries, meta} <- ActivityLog.list_activity_log(params) do
+    {current_function, _} = __ENV__.function
```

Comment:
Please use the `:get_activity_log` as an atom and use it as parameter to the bodyguard policy.

If you want to be more `phoenixish` you can use `https://hexdocs.pm/phoenix/Phoenix.Controller.html#action_name/1` to retrieve the action name.